docs/sample_inventories/migration_sample.yml

```diff
@@ -4,8 +4,9 @@
 ## steps to run:
 ## 1. Run command "ansible-playbook -i <inv.yml> confluent.platform.all" to setup a ZK cluster
 ## 2. Once ZK cluster setup is complete, uncomment "kraft_migration: true" and kafka_controller section.
-## 3. Run command "ansible-playbook -i <inv.yml> confluent.platform.ZKtoKraftMigration"
-## 4. Once Kraft cluster is up and running, Remove "kraft_migration: true" and Zookeeper section from the inventory file.
+## 3. If your sasl_protocol is scram or scram256, uncomment the kafka_controller_sasl_protocol variable and set it to "plain"
```

Comment:
instead of modifying the current sample, please create a new sample inv file for scram migration. We need both types of examples for customers

Reply:
also need to mention that they can set it to kerberos as well but need to setup their kerberos server accordingly in that case

Reply:
added